src/styles/watch.scss

```diff
@@ -0,0 +1,51 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: 80vmin;
```

Comment:
Use 'vw' or 'vh' instead of 'vmin' for width and height to ensure proper scaling on different screen sizes.